### PR TITLE
[MERGE] environment create and object into development.

### PR DESCRIPTION
**Pull request by LotteBase**
This feature adds a environment overview page, a create environment window and the environment scriptable object from which creator options can be called.

### CODE REVIEW

**Review by Vanclyf — CHANGES_REQUESTED** [review]
Remove daan's head from the creator window

**Review by Vanclyf — COMMENTED** [review]
Maybe resize the creatorwindow so you can see all the buttons clearly on the environments tab

**Review by Vanclyf — CHANGES_REQUESTED** [review]
Remove errorlog when first creating an environment
image

**Review by Vanclyf — CHANGES_REQUESTED** [review]
Remove debug logs for images 
image

File: Ravel-Creators/Assets/00-Ravel/01-Scripts/Backend/DataTypes/Organisation.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+public class Organisation : DataContainer
+{
+    public override string Key {
+        get { return organizationId; }
+    }
+    
+    [SerializeField] public string organizationId;
+    [SerializeField] public string organizationName;
+    
+    public Organisation()
```
Comment (Vanclyf) [inline Ravel-Creators/Assets/00-Ravel/01-Scripts/Backend/DataTypes/Organisation.cs:17]: Is an empty constructor necessary?

File: Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/Branding/RavelBranding.cs [hunk 2]
```diff
@@ -3,11 +3,29 @@
...
+    [Tooltip("This is the default banner that is shown in the top of editor windows.")]
     public Texture2D banner;
-    [Tooltip("Zooms in on this point of the banner, use decimals range 0.0 t/m 1.0")]
+    [Tooltip("Zooms in on this point of the banner, use decimals range 0.0 t/m 1.0.")]
     public Vector2 bannerPOI = new Vector2(0.5f, 0.5f);
+    [Tooltip("This default transparant logo is drawn over other images, used as banners.")]
+    public Texture2D overlayLogo;
     
+    [Tooltip("I mean this is self explanatory...")]
     public Texture2D daan;
```
Comment (Vanclyf) [inline Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/Branding/RavelBranding.cs]: Remove me

File: Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/Environment/Environment.cs [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+/// Data class for environments, this data is pulled from the server and put into a scriptable object as container class.
+/// </summary>
+[Serializable]
+public class Environment
+{
+    [ReadOnly]
+    public string environmentUuid;
+    
+    [ReadOnly]
+    public new string name;
```
Comment (Vanclyf) [inline Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/Environment/Environment.cs:18]: new?

File: Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/RavelEditor.cs [hunk 4]
```diff
@@ -1,15 +1,27 @@
...
+    
+    private static bool _retrievingOrganisations;
+    private static Action<Organisation[], bool> _onOrgsRetrieved;
+    
+    /// <summary>
+    /// This object contains custom settable branding files for the editor tools.
+    /// </summary>
     public static RavelBranding Branding {
         get {
             if (_branding == null) {
```
Comment (Vanclyf) [inline Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/RavelEditor.cs:27]: Throw error when there is no config

**Review by Vanclyf — CHANGES_REQUESTED** [review]

File: Ravel-Creators/Assets/00-Ravel/01-Scripts/Backend/DataTypes/Organisation.cs (hunk 1 above)
Comment (LotteBase) [reply Ravel-Creators/Assets/00-Ravel/01-Scripts/Backend/DataTypes/Organisation.cs:17]: This is code from ravel, so I don't want to change it. In ravel the empty one is used there for json serialization

File: Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/Environment/Environment.cs (hunk 3 above)
Comment (LotteBase) [reply Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/Environment/Environment.cs:18]: Name already exists, this prevents unity from throwing a warning for a duplicate varname

File: Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/RavelEditor.cs (hunk 4 above)
Comment (LotteBase) [reply Ravel-Creators/Assets/Ravel-Creators/Scripts/Editor/RavelEditor.cs:27]: Error staat er in, maar de bedoeling is dat dit onze file blijft en niet degene van users denk ik. Maarja better be sure

**Review by Vanclyf — APPROVED** [review]

**Pull request merged**